openassessment/workflow/models.py

```diff
@@ -87,10 +87,7 @@ class AssessmentWorkflow(TimeStampedModel, StatusModel):
     # This Django setting is a list of assessment steps (defined in `settings.ORA2_ASSESSMENTS`)
     # in descending priority order.
     DEFAULT_ASSESSMENT_SCORE_PRIORITY = ['peer', 'self']  # pylint: disable=invalid-name
-    ASSESSMENT_SCORE_PRIORITY = getattr(
```

Comment:
@ahmedbilal Why were these changes introduced? Is this an ORA bug?

Reply:
Please see the below comment.